docs/author/md-fragment.mdx

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Is the frontmatter needed? Just to exclude it from the sidebar.

Maybe we need a naming convention to prevent the other plugins from picking up content fragments. or a `contentType` in the frontmatter?

One for later...